proxy-libs/helioslambda/go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/helios/go-sdk/proxy-libs/helioslambda
+
+go 1.18
```

Comment:
it's 1.18 on purpose? our default is 1.19

Reply:
Our default should be 1.18 otherwise 1.18 go modules won't compile with us. Where do we use 1.19?